### PR TITLE
Cw 679 add ledger litecoin support

### PR DESCRIPTION
**Pull request by konstantinullrich**
# Description

Add LTC Support for Ledger

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [x] Double check modified code and verify it with the feature/task requirements
- [x] Format code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

File: lib/src/screens/connect_device/connect_device_page.dart [hunk 1]
```diff
@@ -35,91 +35,104 @@ class ConnectDevicePage extends BasePage {
...
+  late Timer? _bleStateTimer = null;
   late StreamSubscription<LedgerDevice>? _bleRefresh = null;
 
   @override
   void initState() {
     super.initState();
     WidgetsBinding.instance.addPostFrameCallback((_) {
-      _bleRefreshTimer = Timer.periodic(Duration(seconds: 1), (_) => _refreshBleDevices());
+      _bleStateTimer = Timer.periodic(
+          Duration(seconds: 1), (_) => widget.ledgerVM.updateBleState());
```
Comment (OmarHatem28) [inline lib/src/screens/connect_device/connect_device_page.dart:68]: what does updateBleState do?
and what would happen if ble permission is not granted? would it throw an exception that we should catch here?

File: lib/src/screens/send/send_page.dart
Comment (OmarHatem28) [inline lib/src/screens/send/send_page.dart:1]: revert the formatting here please, I can't review it like this

**Review by OmarHatem28 — CHANGES_REQUESTED** [review]
wish you stop making a lot of reformatting changes 😅

File: cw_bitcoin/lib/electrum_wallet.dart [hunk 2]
```diff
@@ -127,15 +135,26 @@ abstract class ElectrumWalletBase
...
+      Bip32Slip10Secp256k1.fromSeed(seedBytes).derivePath("m/44'/145'/0'")
+          as Bip32Slip10Secp256k1;
 
   static int estimatedTransactionSize(int inputsCount, int outputsCounts) =>
       inputsCount * 68 + outputsCounts * 34 + 10;
 
+  static Bip32KeyNetVersions? getKeyNetVersion(BasedUtxoNetwork network) {
+    switch (network) {
+      case LitecoinNetwork.mainnet:
+        return Bip44Conf.litecoinMainNet.altKeyNetVer;
```
[the remark this answers is not in the document]
Comment (konstantinullrich) [reply cw_bitcoin/lib/electrum_wallet.dart:152]: An xpub has a version code encoded within, a Bitcoin xpub can be used for Litecoin. But it is more common to use an xpub starting with Ltub in litecoin. Bitcoin base calls the Ltub prefix altKeyNetVer, I assume for compatibility reasons.

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**